public/images/sensor-page-header-pixels-mobile.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="336px" height="84px" viewBox="0 0 336 84" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Artboard</title>
```

Comment:
There is some unnecessary stuff in the SVGs. You used Sketch, right? If so, there is a plugin called _SVGO Compressor_ that nicely removes everything that is unnecessary and gives a nice, clean SVG when exporting from Sketch.

Reply:
True! I normally use it but I forgot for this file. It will clean it! Thx! 🙏